lib/active_model_serializers.rb

```diff
@@ -1,32 +1,22 @@
 require 'active_model'
 require 'active_support'
-require 'action_controller'
-require 'action_controller/railtie'
+require 'active_support/core_ext/object/with_options'
 module ActiveModelSerializers
-  mattr_accessor(:logger) { ActiveSupport::TaggedLogging.new(ActiveSupport::Logger.new(STDOUT)) }
-
-  def self.config
-    ActiveModel::Serializer.config
-  end
-
   extend ActiveSupport::Autoload
   autoload :Model
   autoload :Callbacks
   autoload :Deserialization
   autoload :Logging
-end
 
-require 'active_model/serializer'
-require 'active_model/serializable_resource'
-require 'active_model/serializer/version'
+  class << self; attr_accessor :logger; end
+  self.logger = ActiveSupport::TaggedLogging.new(ActiveSupport::Logger.new(STDOUT))
```

Comment:
I think the `mattr_accessor(:logger) { ActiveSupport::TaggedLogging.new(ActiveSupport::Logger.new(STDOUT)) }` is more legible than this one.